### ISSUE TITLE
[BUG] i cant see my list of games, although they are supporting DLSS

### ISSUE THREAD

**Issue by omarmoha93** [beeradmoore/dlss-swapper#186]
**Describe the bug**
i can only see two games one with Origin  "the finals"
other one is a steam one "warhaven"

other games like BF 2042 , lords of the fallen are not showing in the list

**Comment by beeradmoore** [comment]
Hey @omarmoha93 ,
A few things to check.

Are you using the latest version of DLSS Swapper? Check the releases tab on this repo to see the latest releases.

On the games page there is a filter in the top right, is it set to hide non-DLSS titles?

**Comment by omarmoha93** [comment]
hello and thanks for your efforts <3 

yes i`m using latest version 1.0.3.1, and i`m aware of the filter, unfortunately still BF and Lords of the fallen are not showing in the list although they are DLSS compatible

**Comment by beeradmoore** [comment]
No worries, I am here to help! What games marketplace did install BF2024 and Lords of the Fallen from?

**Comment by omarmoha93** [comment]
BF Origin
LOTF steam

note that i have two SSDs idk if this info might be useful

**Comment by beeradmoore** [comment]
Thanks for the info. 

Multiple drives are supported so it shouldn't be that.

Before we look into other things, are any other Steam or Origin games showing?

Trying to think of ways our PC could be different, do you have its (or steams) language set to English or something else?

**Comment by dkflint723** [comment]
@beeradmoore, I'm just pinging onto this ticket since I'm having the same problem. For some reason, my Steam games just don't show up sometimes. Occasionally, all or just some will show up, but typically, none are detected. I'm currently on 1.1.5.

**Comment by beeradmoore** [comment]
@dkflint723 , can you go to settings and change the logging to verbose, and then restart DLSS Swapper, once it has loaded the games hit refresh, and once again wait for it to load games. 

After all that go back to settings where it says your current log, and then upload that to a reply here. If there is something funky happening ideally that would show it.

**Comment by dkflint723** [comment]
Here's a copy of my logs from today. I did at one point end up uninstalling and reinstalling steam. Originally it was on my C: drive, but I moved it to my D: drive where I store my games. Since doing this I haven't had anymore issues with the games not showing up as intended.

[dlss_swapper_20250215.log](https://github.com/user-attachments/files/18813395/dlss_swapper_20250215.log)

**Comment by beeradmoore** [comment]
Yeah nothing there seems to show Steam games not loading. Just that Ubisoft Connect is not installed (or if it is it can't find any games), and that it can't read data for Avowed (XboxApp). Maybe the game is open, maybe Windows is protecting the files.

in either case it seems that you had fixed your issue, and OP has not responded in over a month so I am closing this for now.
